game/lib/global_leaderboard/global_leaderboard_widget.dart

```diff
@@ -83,132 +93,129 @@ class _GlobalLeaderboardWidgetState extends State<GlobalLeaderboardWidget> {
+                    // Use this line below to retrieve actual data
+                    final List<LeaderDto> list =
+                        myEventModel.getTopPlayersForEvent('', 1000);
+                    // Leaderboard starts at 4th position because first three already in podium
+                    int position = 4;
+
+                    list.sort((a, b) => b.score.compareTo(a.score));
+
+                    return Container(
+                      width: 360.0,
+                      height: 446.0,
+                      decoration: BoxDecoration(
+                        color: Color.fromRGBO(255, 170, 91, 0.15),
+                        borderRadius: BorderRadius.only(
+                          topLeft: Radius.circular(10.0),
+                          topRight: Radius.circular(10.0),
                         ),
-                        child: Container(
-                          width: 283.05,
-                          height: 432.0,
-                          child: ListView(
-                            shrinkWrap: true,
-                            scrollDirection: Axis.vertical,
-                            children: [
-                              for (LeaderDto user in list.skip(3))
-                                Padding(
-                                  padding: const EdgeInsets.only(
-                                      left: 30.95, right: 30.95, top: 16.0),
-                                  child: Container(
-                                    decoration: BoxDecoration(
-                                      borderRadius: BorderRadius.only(
-                                        topLeft: Radius.circular(10.0),
-                                        topRight: Radius.circular(10.0),
-                                        bottomLeft: Radius.circular(10.0),
-                                        bottomRight: Radius.circular(10.0),
-                                      ),
-                                      color: Colors.white,
-                                      boxShadow: [
-                                        BoxShadow(
-                                          color: Color(0x40000000),
-                                          offset:
-                                              Offset(0.0, 1.7472529411315918),
-                                          blurRadius: 6.989011764526367,
-                                        ),
-                                      ],
-                                    ),
-                                    child: leaderBoardCell(
-                                      context,
-                                      user.username,
-                                      position++,
-                                      user.score,
-                                      user.userId == myUserModel.userData?.id,
+                      ),
+                      child: Container(
+                        width: 283.05,
+                        height: 432.0,
```

Comment:
Set the width and height to be a percentage of the total screen instead of hard coding values. For example, if you want the width to take up 90% of the screen width, you can do width: MediaQuery.of(context).size.width * 0.9.